src/data/markdown/docs/03 cloud/03 Integrations/05 Cloud APM.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Cloud APM'
+excerpt: 'How to export metrics from k6 Cloud to APM platforms'
+---
+
+A new feature has been recently added to the k6 Cloud service, for users to be able to export metrics from a running test on k6 Cloud in near real-time to their preferred [APM](https://en.wikipedia.org/wiki/Application_performance_management) platform(s).
+
+## Supported APM Providers
+
+Currently, the following platforms are supported:
+
+| Provider  | URL(s)                      |
+| --------- | --------------------------- |
+| DataDog   | <https://www.datadoghq.com> |
+| DataDogEU | <https://www.datadoghq.eu>  |
+
+## Cloud APM Configuration
+
+For configuring each platform on your load test on k6 Cloud, you should add your desired APM configuration to the [extension options](/using-k6/options#extension-options) section of your load test script. The configuration parameters are as follows:
+
+| Name                     | Description                                                                                                                           |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------- |
+| `provider`               | Any APM provider name available in the [supported APM provider](#supported-apm-providers)'s table.                                    |
+| `api_key`                | The `api_key` provided by the APM platforms.                                                                                          |
+| `app_key`                | The `app_key` provided by the APM platforms, if applicable.                                                                           |
+| `export_metrics`         | List of custom metrics to be exported.                                                                                                |
```

Comment:
Maybe we should rename this parameter to just `metrics`? 
What do you think?

Reply:
How about `custom_metrics`?

Reply:
Do we mean "custom metrics" (as in https://k6.io/docs/using-k6/metrics#custom-metrics) or just any metric?

Reply:
@robingustafsson Exactly, custom metrics.

Reply:
Hmm, is it only for custom metrics though? 
Is the user not able to specify some of the built-in metrics that we don't export by default? 
For example `checks` or `http_req_sending` ?

Reply:
They can.

Reply:
I'll rephrase sentences to include built-in and custom metrics and will rename the config variable to `metrics`.